packages/core/data_adapters/BaseAdapter.ts

```diff
@@ -108,6 +112,10 @@ export abstract class BaseFeatureDataAdapter extends BaseAdapter {
   //   const { refNames } = this.metadata
   //   return refNames
   // }
+  //
+  getConf(arg: string | string[]) {
```

Comment:
helper function for readConfObject on the adapter, since config is stored on the adapter's this.config...